test/net/sourceforge/kolmafia/request/AdventureRequestTest.java

```diff
@@ -30,6 +35,12 @@ public class AdventureRequestTest {
   public void init() {
     KoLCharacter.reset("AdventureRequestTest");
     Preferences.reset("AdventureRequestTest");
+    AdventureQueueDatabase.allowSerializationWrite = false;
+  }
+
+  @AfterEach
+  public void restore() {
```

Comment:
could be `AfterAll`. No test wants this to be true.

Reply:
I was mirroring the BeforeEach.  If I set it BeforeEach then I should clear it AfterEach but I don't think it matters until a test wants to write the file.

Reply:
I suppose I think of it as a performance thing: with before each / after each it sets and unsets it every test. With before each / after all it sets it every test (perhaps unnecessarily), but only unsets it at the end: fewer actions.